src/js/player.js

```diff
@@ -2941,92 +2739,144 @@ Player.prototype.options_ = {
+   *
+   * @return {Boolean} True if the player is in the seeking state, false if not.
+   * @method Player.prototype.seeking
+   */
+  'seeking',
+  /**
+   * Returns the TimeRanges of the media that are currently available
+   * for seeking to.
+   *
+   * @return {TimeRanges} the seekable intervals of the media timeline
+   * @method Player.prototype.seekable
+   */
+  'seekable',
+  /**
+   * Returns the current state of network activity for the element, from
+   * the codes in the list below.
+   * - NETWORK_EMPTY (numeric value 0)
+   *   The element has not yet been initialised. All attributes are in
+   *   their initial states.
+   * - NETWORK_IDLE (numeric value 1)
+   *   The element's resource selection algorithm is active and has
+   *   selected a resource, but it is not actually using the network at
+   *   this time.
+   * - NETWORK_LOADING (numeric value 2)
+   *   The user agent is actively trying to download data.
+   * - NETWORK_NO_SOURCE (numeric value 3)
+   *   The element's resource selection algorithm is active, but it has
+   *   not yet found a resource to use.
+   *
+   * @see https://html.spec.whatwg.org/multipage/embedded-content.html#network-states
+   * @return {Number} the current network activity state
+   * @method Player.prototype.networkState
+   */
+  'networkState',
+  /**
+   * Returns a value that expresses the current state of the element
+   * with respect to rendering the current playback position, from the
+   * codes in the list below.
+   * - HAVE_NOTHING (numeric value 0)
+   *   No information regarding the media resource is available.
+   * - HAVE_METADATA (numeric value 1)
+   *   Enough of the resource has been obtained that the duration of the
+   *   resource is available.
+   * - HAVE_CURRENT_DATA (numeric value 2)
+   *   Data for the immediate current playback position is available.
+   * - HAVE_FUTURE_DATA (numeric value 3)
+   *   Data for the immediate current playback position is available, as
+   *   well as enough data for the user agent to advance the current
+   *   playback position in the direction of playback.
+   * - HAVE_ENOUGH_DATA (numeric value 4)
+   *   The user agent estimates that enough data is available for
+   *   playback to proceed uninterrupted.
+   *
+   * @see https://html.spec.whatwg.org/multipage/embedded-content.html#dom-media-readystate
+   * @return {Number} the current playback rendering state
+   * @method Player.prototype.readyState
+   */
+  'readyState'
+].forEach(function(fn) {
+  Player.prototype[fn] = function() {
+    return this.techGet_(fn);
+  };
+});
+
+TECH_EVENTS_RETRIGGER.forEach(function(event) {
+  Player.prototype[`handleTech${toTitleCase(event)}_`] = function() {
+    return this.trigger(event);
+  };
+});
+
+/* document methods *?
 /**
  * Fired when the player has initial duration and dimension information
  *
  * @event loadedmetadata
+ * Player.prototype.handleLoadedMetaData_
```

Comment:
We probably don't want to have `@method` here. We don't want these to show up as documentation. These stubs specifically are used to document the events that the player listens to but are otherwise private methods. Adding a `@private`, I think, will then prevent the event from being displayed in the docs.